### ISSUE TITLE
Publish helm artifacts to a repository

### ISSUE THREAD

**Issue by deansheather** [coder/coder#3127]
Once #2746 is merged, we will have helm artifact packaging, but the resulting packages will only upload to GitHub releases. In Coder Classic we upload our artifacts to `https://helm.coder.com` so they can be used with `helm repo add`.

We need to determine where we want to publish our packages for Coder OSS and/or Coder V2 Enterprise and what they will be called, then update `./scripts/helm.sh` to do the thing.

**Comment by deansheather** [comment]
@kylecarbs do you have any thoughts on where we should publish these? I looked into publishing them as OCI artifacts to github packages but it seems like it's not supported yet and I honestly couldn't get the helm CLI to play along with any of the OCI packaging stuff

**Comment by kylecarbs** [comment]
Closing in favor of #3545.

@deansheather sorry for the slow response here! I think the user could even be directed to clone our repo and run `helm install ./coder` if that's possible!
